gluqlo.c

```diff
@@ -37,7 +37,7 @@ const char* TITLE = "Gluqlo 1.1";
 const int DEFAULT_WIDTH = 1024;
 const int DEFAULT_HEIGHT = 768;
 
-bool twentyfourh = true;
+bool twentyfourh = false;
```

Comment:
Not sure if this really is required to install for Fedora. And personally I prefer 24h format. But I'm nobody here, so never mind :)